### PR TITLE
REL: preliminaries

### PR DESCRIPTION
**Pull request by wasade**
Partially addresses #50 

Note, we are registered on `testpypi` and `python setup.py sdist` appears to package as expected.

### CODE REVIEW

File: scripts/deblur [hunk 1]
```diff
@@ -453,13 +453,11 @@ def build_biom_table(seqs_fp, output_biom_fp, min_reads, file_type, log_level, l
               show_default=True,
               help="indicates this is not the main deblur process "
               "(used by the parallel deblur - do not use manually)")
-@click.option('attack_mode', '-A', required=False, type=bool, default=False,
-              help="ATTACK MODE (http://xkcd.com/1692)")
```
Comment (amnona) [inline scripts/deblur]: Every OTU picking method needs an Attack mode!!!!
(would prefer to keep it :) )

**Comment by amnona** [comment]
@wasade - there are some merge conflicts.
Also, not all comments have been addressed ;)

**Comment by wasade** [comment]
It's not an easter egg if it's in your face...

**Comment by amnona** [comment]
not supposed to be an easter egg but rather an easter bunny :)
i like having it in the --help. do you think we can keep it?

**Comment by wasade** [comment]
its back

File: ChangeLog.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# deblur changelog
+
+## Version 0.1.0-dev  (changes since 0.0.1 go here)
```
Comment (mortonjt) [inline ChangeLog.md]: `(changes since 0.0.1 go here)` -> `(changes since 0.1.0 go here)`

**Comment by mortonjt** [comment]
other than the 1 typo, I think it is good to go

**Comment by wasade** [comment]
I'd say its more accurate to just omit that "since" stuff since there hasn't been a release

**Comment by mortonjt** [comment]
👍

**Comment by mortonjt** [comment]
Looks like there is a flake8 error ...

**Comment by amnona** [comment]
@wasade want to merge but it's failing (pep8...)

**Comment by amnona** [comment]
getting close
for deblur 2.0:
-b --suppress-bees :)

**Pull request merged**